app/models/phase.rb

```diff
@@ -58,11 +58,12 @@ def finished?
   end
 
   def in_progress?
-    ((self.initial_date <= Time.zone.now) and (self.final_date >= Time.zone.now)) && self.plugin_relation.ready?
+    ((self.initial_date <= Time.zone.now) and (self.final_date >= Time.zone.now)) &&
+      (self.plugin_relation.present? && self.plugin_relation.ready?)
```

Comment:
IMO we should not check plugin_relation existence. If we don't hava a plugin relation, something is very wrong with the system.